Services/UserService/Utils/TokenMiddleware.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using UserService.Properties;
+
+namespace UserService.Utils
+{
+    public class TokenMiddleware
+    {
+        private RequestDelegate next;
+
+        public TokenMiddleware(RequestDelegate next)
+        {
+            this.next = next;
+        }
+        
+
+        public async Task InvokeAsync(HttpContext context)
+        {
+            if (context.Request.Method == "POST" & string.Compare(context.Request.Path, StringCons.UrlCreateUser, StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
Not convenient way to read , it is better to wrap text in two lines

Reply:
fixed